database/src/models/serializers.rs

```diff
@@ -82,17 +82,20 @@ pub(crate) fn extract_action_type_and_value_from_action_view(
         } => {
             let mut arguments = json!({
                 "method_name": method_name.escape_default().to_string(),
-                "args_base64": base64::encode(&args),
+                "args_base64": args,
                 "gas": gas,
                 "deposit": deposit.to_string(),
             });
 
             // During denormalization of action_receipt_actions table we wanted to try to decode
-            // args in case if it is a JSON object and put them along side with args_base64
+            // args which is base64 encoded in case if it is a JSON object and put them near initial
+            // args_base64
             // See for reference https://github.com/near/near-indexer-for-explorer/issues/87
-            if let Ok(mut args_json) = serde_json::from_slice(&args) {
-                escape_json(&mut args_json);
-                arguments["args_json"] = args_json;
+            if let Ok(decoded_args) = base64::decode(args) {
+                if let Ok(mut args_json) = serde_json::from_slice(&decoded_args) {
+                    escape_json(&mut args_json);
+                    arguments["args_json"] = args_json;
+                }
```

Comment:
This change looks dangerous in this PR. I mean, all the other changes are all about changing the structure, library, or data source. This one changes the logic. 
If we have a bug in this, it would be hell to extract the bug from this huge PR.
Let's move this change to a separate PR?

Reply:
The thing is, this project is not going to work without this change, because atm it uses an older version of `near-lake-framework` and that one returns this data in base64-encoded String, while newer version (nearcore as well returns raw `Vec<u8>`. So it should be here, and then it should be changed within the upgrade to `0.6.0` lake-framework

Reply:
Resolving as this change is required